src/Units/ArmyWrappers/ZerglingWrapper.cpp

```diff
@@ -91,9 +91,14 @@ void ZerglingWrapper::onFrame() {
 }
 
 void ZerglingWrapper::displayInfo() {
-	if (this->unit->getLastCommand().getTarget()) {
-		BWAPI::Broodwar->drawLineMap(this->unit->getPosition(), this->unit->getLastCommand().getTarget()->getPosition(), BWAPI::Colors::White);
-	} else if (this->unit->getLastCommand().getTargetPosition().isValid()) {
-		BWAPI::Broodwar->drawLineMap(this->unit->getPosition(), this->unit->getLastCommand().getTargetPosition(), BWAPI::Colors::White);
-	}
+    if (this->unit->getLastCommand().getTarget()) {
+        BWAPI::Broodwar->drawLineMap(this->unit->getPosition(), this->unit->getLastCommand().getTarget()->getPosition(), BWAPI::Colors::White);
+    }
+    else if (this->unit->getLastCommand().getTargetPosition().isValid()) {
+        BWAPI::Broodwar->drawLineMap(this->unit->getPosition(), this->unit->getLastCommand().getTargetPosition(), BWAPI::Colors::White);
+    }
+    else if (this->unit->getLastCommand().getTargetTilePosition().isValid()) {
```

Comment:
I believe the `getTargetPosition` and `getTargetTilePosition` should return the same thing, just different representation(the first being a `BWAPI::Position`, the second being a `BWAPI::TilePosition`). The first is more specific, so if they both do the same thing, only use that.